packages/ui/lib/components/StreamInfo/index.js

```diff
@@ -52,6 +53,11 @@ const StreamInfo = ({
         )}
       </div>
       <div className={styles.stream_buttons}>
+        {selectedStream.originalUrl && (
+          <a href='#' data-testid='copy-orignal-url' onClick={handleCopyStreamUrl}>
+            <Icon name='copy' />
```

Comment:
this icon suggests copying a file, better user `linkify`, and add a tooltip saying it copies the link to your clipboard